src/DynamoRevit/ViewModel/RevitWatch3DViewModel.cs

```diff
@@ -70,7 +70,7 @@ protected override void OnActiveStateChanged()
 
         protected override void OnEvaluationCompleted(object sender, EvaluationCompletedEventArgs e)
         {
-            Draw();
+            if (e.EvaluationTookPlace) Draw();
```

Comment:
does this handle all cases? i.e. if I delete a node will the geometry in Revit preview be deleted?